randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -365,102 +577,182 @@ Extra - asset_id: collision_camera_009
+  > Dock to Super Missile Room
+      Trivial
 
 > Tile Group (BOMB); Heals? False
   * Configurable Node
   * Extra - actor_name: breakabletilegroup_019
   * Extra - actor_def: actordef:actors/logic/breakabletilegroup/charclasses/breakabletilegroup.bmsad
   * Extra - actor_layer: breakables
   * Extra - tile_types: ('BOMB',)
+  > Door to collision_camera_008 (G)
+      Trivial
+  > Bomb Ledge
+      Can Slide
+
+> Dock to Super Missile Room; Heals? False
+  * Morph Ball Tunnel to Super Missile Room/Dock to collision_camera_009 (G)
+  > Pickup (item_missiletankplus_000)
+      Morph Ball
+
+> Dock to Spider Magnet Elevator; Heals? False
+  * Morph Ball Tunnel to Spider Magnet Elevator/Dock to collision_camera_009 (G)
+  > Dock to collision_camera_008 (G)
+      Trivial
+
+> Dock to collision_camera_008 (G); Heals? False
+  * Slide Tunnel to collision_camera_008 (G)/Dock to collision_camera_009 (G)
+  > Dock to Spider Magnet Elevator
+      Trivial
+
+> Bomb Ledge; Heals? False
+  > Door to Super Missile Room (Missile Lock)
+      Trivial
+  > Tile Group (BOMB)
+      Morph Ball
 
 ----------------
-collision_camera_010 (G)
+Super Missile Room
 (Valid Starting Location)
 Extra - total_boundings: {'x1': -5000.0, 'x2': -2900.0, 'y1': -6000.0, 'y2': -3900.0}
 Extra - polygon: [[-2900.0, -3900.0], [-5000.0, -3900.0], [-5000.0, -6000.0], [-2900.0, -6000.0]]
 Extra - asset_id: collision_camera_010
 > Door to collision_camera_001 (G); Heals? False
-  * Power Beam Door to collision_camera_001 (G)/Door to collision_camera_010 (G)
+  * Power Beam Door to collision_camera_001 (G)/Door to Super Missile Room
   * Extra - actor_name: doorpowerpower_012
   * Extra - actor_def: actordef:actors/props/doorpowerpower/charclasses/doorpowerpower.bmsad
   * Extra - left_shield_entity: {EMPTY}
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
+  > Door to collision_camera_009 (G) (Super Lock)
+      Trivial
 
-> Dock to collision_camera_009 (G) (doorclosedpower_001); Heals? False
-  * Not Determined to collision_camera_009 (G)/Dock to collision_camera_010 (G) (doorclosedpower_001)
+> Door to collision_camera_009 (G) (Super Lock); Heals? False
+  * Power Beam Door to collision_camera_009 (G)/Door to Super Missile Room (Super Lock)
   * Extra - actor_name: doorclosedpower_001
   * Extra - actor_def: actordef:actors/props/doorclosedpower/charclasses/doorclosedpower.bmsad
   * Extra - left_shield_entity: {EMPTY}
   * Extra - left_shield_def: None
   * Extra - right_shield_entity: Root:pScenario:rEntitiesLayer:dctSublayers:default:dctActors:doorshieldsupermissile_002
   * Extra - right_shield_def: actordef:actors/props/doorshieldsupermissile/charclasses/doorshieldsupermissile.bmsad
+  > Door to collision_camera_001 (G)
+      Trivial
+  > Dock to collision_camera_009 (G)
+      All of the following:
+          Morph Ball
+          Speed Booster or Simple IBJ or Use Spin Boost
```

Comment:
You can also walljump and flash shift, but I guess that's more of a trick.

Reply:
added